components/organisms/messaging/send/send.js

```diff
@@ -7,7 +7,7 @@ import React, { useState, useRef } from 'react'
 import { number, func, array } from 'prop-types'
 
 // UI
-import { Button, EmojiMart, Form, Icon, Textarea, Dropdown } from '../../../../'
+import { Button, EmojiMart, Form, Icon, Textarea, Dropdown } from '../../../'
```

Comment:
I have trained my self so hard about this alpha thing :)

Reply:
@Ali-sharafi-git Ha and now you spot it easily cause it sticks out :) seriously saves time when scanning code.
I however did not spot it when I was refactoring it.... next time.